openmetadata-ui/src/main/resources/ui/src/components/common/Table/Table.tsx

```diff
@@ -22,22 +23,20 @@ const Table = <T extends object = any>({ loading, ...rest }: TableProps<T>) => {
 
   if (isLoading) {
     const { columns } = { ...rest };
-    const dataSource = [...Array(3)].map((_, index) => ({
-      key: `key${index}`,
+    const dataSource = [...Array(3)].map(() => ({
+      key: `key${uniqueId()}`,
```

Comment:
@ShaileshParmar11 I asked to use value from map itself
```suggestion
    const dataSource = [...Array(3)].map((key) => ({
      key: key,
```